teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/DebugDbCommand.java

```diff
@@ -207,10 +207,7 @@ public int getBlockCounts(
       try (Stream<SignedBeaconBlock> stream = database.streamHotBlocks()) {
         printIfPresent("Hot blocks", stream.count());
       }
-      try (Stream<SignedBeaconBlock> stream =
-          database.streamFinalizedBlocks(UInt64.ZERO, UInt64.MAX_VALUE)) {
-        printIfPresent("Finalized blocks", stream.count());
-      }
+      printIfPresent("Finalized blocks", database.countUnblindedFinalizedBlocks());
```

Comment:
this count got added because it is significantly faster than streaming actual signed blocks.. the count uses raw streams, so that has been kept low level.